src/app/components/turnos/dashboard/estadisticas-pacientes.component.ts

```diff
@@ -69,6 +73,15 @@ export class EstadisticasPacientesComponent implements OnInit {
             nroCarpeta: ''
         };
         this.getPaciente();
```

Comment:
Evitemos usar _id en la aplicación. Esto es algo interno de MongoDB. Usemos siempre id.
Por ejemplo...
organizacion: {
                 _id: this.auth.organizacion.id
...
}

debería ser...
organizacion: {
                 id: this.auth.organizacion.id,	
....
}